policyengine_us/variables/gov/states/de/tax/income/deductions/de_tax_unit_itemizes.py

```diff
@@ -0,0 +1,14 @@
+from policyengine_us.model_api import *
+
+
+class de_tax_unit_itemizes(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Whether the tax unit in Delaware itemizes the deductions when married filing separately"
```

Comment:
does this only apply to separate on same return? is there another variable for the choice in other filing statuses?

Reply:
we use the same variable in all cases, the label is misleading, changed it